src/Console/Command/SelfUpdateCommand.php

```diff
@@ -69,36 +69,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        $remoteTag = $this->getLatestTag();
-        if (null === $remoteTag) {
-            $output->writeln('<error>Unable to determine newest version.</error>');
+        $checker = new NewVersionChecker('v1.1.0');
 
-            return 0;
-        }
-
-        $currentVersion = 'v'.$this->getApplication()->getVersion();
-        if ($currentVersion === $remoteTag) {
+        $latestVersion = $checker->getLatestVersion();
```

Comment:
previously, there were nice error msg if fetching from github fails (and it may happen!)
> $output->writeln('<error>Unable to determine newest version.</error>');

now, we just have app crashed

Reply:
Error handling restored.